apps/apps.tf

```diff
@@ -1,7 +0,0 @@
-module "app_bookinfo" {
-  source                     = "./modules/apps/bookinfo"
-  cluster_name               = local.cloud[var.cloud][var.cluster_id].cluster_name
-  k8s_host                   = local.cloud[var.cloud][var.cluster_id].host
-  k8s_cluster_ca_certificate = local.cloud[var.cloud][var.cluster_id].cluster_ca_certificate
-  k8s_client_token           = local.cloud[var.cloud][var.cluster_id].token
-}
```

Comment:
This was probably a leftover as now it's configured via gitops. Anyway, this is referencing variables that do not exist in this module so it's probably just worth removing it.